### PR TITLE
Remove deprecated Disconnect lists from "defaults"

### PR DESCRIPTION
**Pull request by PromoFaux**
**By submitting this pull request, I confirm the following:** 

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes, and have included unit tests where possible.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
**What does this PR aim to accomplish?:**

See #3450 for details

Removing these lists only removes 337 unique domains from a fresh install with all the lists selected. No brainer really, especially as they're deprecated!

With:
```
[i] Number of gravity domains: 87259 (84818 unique domains)
```
Without:
```
[i] Number of gravity domains: 84524 (84481 unique domains)
```

### CODE REVIEW

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -1240,8 +1238,6 @@ appendToListsFile() {
     case $1 in
         StevenBlack  )  echo "https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts" >> "${adlistFile}";;
         MalwareDom   )  echo "https://mirror1.malwaredomains.com/files/justdomains" >> "${adlistFile}";;
-        DisconTrack  )  echo "https://s3.amazonaws.com/lists.disconnect.me/simple_tracking.txt" >> "${adlistFile}";;
```
Comment (XhmikosR) [inline automated install/basic-install.sh:1243]: Are both lists deprecated? Just wondering :)

Comment (PromoFaux) [reply automated install/basic-install.sh:1243]: Per the response from disconnect.me to the op of #3450 

>The list you referenced was used in our legacy products. It is not maintained, has not been updated, and is not actively distributed us. Airbrake is not included on our main tracker list (linked below) which is used by partners such as Firefox.

>https://github.com/disconnectme/disconnect-tracking-protection/blob/master/services.json

I'm making the reasonable assumption that both lists are no longer updated/distributed. Besides which, when removed only 337 domains are "lost" from an install that has all lists selected.

**Comment by pralor** [comment]
This pull request has been mentioned on **Pi-hole Userspace**. There might be relevant details there:

https://discourse.pi-hole.net/t/initial-deployment-docker-errors/34811/15

**Comment by pralor** [comment]
This pull request has been mentioned on **Pi-hole Userspace**. There might be relevant details there:

https://discourse.pi-hole.net/t/pi-hole-5-1-released/35577/1

**Pull request merged**